test/git-lfs/scenarios.json

```diff
@@ -0,0 +1,20 @@
+{
+    "autoPullEnabled": {
+        "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+        "initializeCommand": "(git clone https://github.com/codspace/git-lfs-example /tmp/testRepo || true) && (cp -r /tmp/testRepo/.git* . && cp -r /tmp/testRepo/* .)",
```

Comment:
Should I move this to the `devcontainers` org?   Also open to other ideas, but at the end of the day the best way to test this is to have a live git repo cloned in with an LFS artifact.

Reply:
> Should I move this to the devcontainers org?

I like that, I wonder if we should create a new `test` repo for this purpose and add scenarios to different branches. This way we don't have to worry about unwanted dependency on other repos. We'd like to avoid test failures just because `codspace/git-lfs-example` was deleted at some point.